ui/.nvmrc

```diff
@@ -1 +1 @@
-v18.20.2
+v22.13.1
```

Comment:
Should we use just the major version here to avoid frequent changes? Or major and minor, at the very least?

Reply:
Good point, should be just a major version now.